### PR TITLE
Solve pylint issues in tests

### PR DESCRIPTION
**Pull request by MartinHjelmare**

### CODE REVIEW

File: tests/test_command.py [hunk 1]
```diff
@@ -17,24 +15,24 @@ def ec():
...
     assert command.observe_duration == 0
-    assert command.err_callback == ec
+    assert command.err_callback is error_callback
 
 
 def test_result():
     """Test callback process_result."""
 
-    def pr(value):
+    def process_result(value):
```
Comment (ggravlingen) [inline tests/test_command.py:32]:
```suggestion
    def process_result(value: int):
```

Maybe add type hint here to? Not very important though.

File: tests/test_util.py [hunk 2]
```diff
@@ -37,19 +37,20 @@ def tearDown(self):
...
         """Test json load."""
-        f = open(path.join(self.test_dir, "sample_psk_file2.txt"), "w")
         config = {"identity": "hashstring", "key": "secretkey"}
         data = json.dumps(config, sort_keys=True, indent=4)
-        f.write(data)
-        f.close()
+        with open(
+            path.join(self.test_dir, "sample_psk_file2.txt"), "w", encoding="utf-8"
+        ) as fil:
+            fil.write(data)
```
Comment (ggravlingen) [inline tests/test_util.py:53]:
```suggestion
        ) as file:
            file.write(data)
```

File: tests/test_util.py [hunk 3]
```diff
@@ -60,20 +61,21 @@ def test_load_file_not_found(self):
...
     def test_load_not_json(self):
         """Test of load not json."""
-        f = open(path.join(self.test_dir, "sample_psk_file3.txt"), "w")
         data = "{not valid json"
-        f.write(data)
-        f.close()
+        with open(
+            path.join(self.test_dir, "sample_psk_file3.txt"), "w", encoding="utf-8"
+        ) as fil:
+            fil.write(data)
```
Comment (ggravlingen) [inline tests/test_util.py:68]:
```suggestion
        ) as file:
            file.write(data)
```

**Review by ggravlingen — APPROVED** [review]
Just a few minor comments. You have a better sense than me if they really are important edits in the grand scheme of things. Otherwise, this looks good!

File: tests/test_command.py (hunk 1 above)
Comment (MartinHjelmare) [reply tests/test_command.py:32]: I didn't type the tests completely as we don't check that currently.

File: tests/test_util.py (hunk 2 above)
Comment (ggravlingen) [reply tests/test_util.py:53]: Gotcha, let's keep fil.

File: tests/test_command.py (hunk 1 above)
Comment (ggravlingen) [reply tests/test_command.py:32]: Ok!

**Review by ggravlingen — APPROVED** [review]

**Pull request merged**